build.gradle

```diff
@@ -384,6 +386,28 @@ subprojects {
         }
     }
 
+    dependencyCheck {
+        scanConfigurations = [
+            'compile',
+            'implementation',
+            'compileOnly',
+            'compileClasspath',
+            'runtime',
+            'runtimeOnly',
+            'runtimeClasspath',
+            'default',
+        ]
+        analyzers {
+            assemblyEnabled = false
+        }
+        suppressionFile = file("${rootDir}/OWASP-Suppression.xml")
+
+        // Configures the dependencyCheck plugin to fail the build if any vulnerability with a CVSS score over the configured value is found
+        failBuildOnCVSS = 5
```

Comment:
I'm not sure that this is the appropriate level to fail on. The scale is 1-10, so if we put 1 here we will always fail.

Reply:
Turns out the number to always fail when a vulnerability found is actually 0. That being the case, I have set this to 0.